src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -199,7 +201,7 @@ public bool SetNullableContext(byte? value)
             _flags = new Flags(specialType, typeKind);
 
             var containingType = this.ContainingType;
-            if ((object)containingType != null && containingType.IsSealed && this.DeclaredAccessibility.HasProtected())
+            if (containingType?.IsSealed == true && this.DeclaredAccessibility.HasProtected())
```

Comment:
Nit: Our general approach to nullable annotating has been to keep changes minimal. If you're going to go as far as to use `?.`, you may was well go all the way to pattern matching.

Reply:
This is minimal only because the conversion was producing a nullable warning.